js/src/common/helpers/avatar.tsx

```diff
@@ -1,26 +1,28 @@
+import * as Mithril from 'mithril';
+import User from '../models/User';
+
 /**
  * The `avatar` helper displays a user's avatar.
  *
- * @param {User} user
- * @param {Object} attrs Attributes to apply to the avatar element
- * @return {Object}
+ * @param attrs Attributes to apply to the avatar element
  */
-export default function avatar(user, attrs = {}) {
+
```

Comment:
Extra space?

Reply:
Yes, fixed in avatar.tsx and username.tsx.